src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/Detail.tsx

```diff
@@ -288,6 +291,16 @@ export const Detail = () => {
                             </div>
                           )}
                         />
+                        {isTesting && (
+                          <ModalDialog
+                            onClose={handleOnDelete}
+                            title={'om te kunnen testen'}
+                            description={
+                              'This defines the alignment along the main axis. It helps distribute extra free space leftover when either all the flex items on a line are inflexible, or are flexible but have reached their maximum size. It also exerts some control over the alignment of items when they overflow the line.This defines the alignment along the main axis. It helps distribute extra free space leftover when either all the flex items on a line are inflexible, or are flexible but have reached their maximum size. It also exerts some control over the alignment of items when they overflow the line.This defines the alignment along the main axis. It helps distribute extra free space leftover when either all the flex items on a line are inflexible, or are flexible but have reached their maximum size. It also exerts some control over the alignment of items when they overflow the line.'
```

Comment:
Why is this? You should commit testing code to the `main` branch

Reply:
Ah, I did this as a service to you. So that you would have a modal to see if it was all right. Removed it now

Reply:
Removed it